### ISSUE TITLE
[Bug] Firebase Analytics disabled due to missing google_app_id on Android

### ISSUE THREAD

**Issue by okuar** [firebase/firebase-unity-sdk#981]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: 2022.3.6f1.
  * Firebase Unity SDK version: 11.8.1
  * Source you installed the SDK: unitypackage
  * Problematic Firebase Component: Analytics
  * Other Firebase Components in use: none
  * Additional SDKs you are using: IronSource
  * Platform you are using the Unity editor on: Windows
  * Platform you are targeting: Android
  * Scripting Runtime: IL2CPP
  * Pre-built SDK from the [website](https://firebase.google.com/download/unity) or open-source from this repo: Pre-built SDK 

### [REQUIRED] Please describe the issue here:

Firebase Analytics is disabled due to a missing google_app_id. This issue is consistently reproducible and results in Firebase Analytics being disabled, with logs indicating that the app measurement is disabled due to the missing ID.

#### Steps to reproduce:

*Import Firebase Analytics SDK version 11.8.1.

*Configure Firebase according to the setup instructions, including adding the google-services.json file to the project.

*In the custom launcher template, the following packagingOptions were added:

packagingOptions {
pickFirst 'META-INF/kotlinx_coroutines_core.version'
doNotStrip '/armeabi-v7a/.so'
doNotStrip '/arm64-v8a/.so'
}

//i've added it because having multiple copies of the kotlinx_coroutines_core.version file from different dependencies do not allow to make a buildd

* Manually add Firebase configuration in the Unity script, appId, projectId, because otherwise an error occurred during initialization

*Build and run the app on an Android device.

* Checked with logcat:
04-03 19:51:17.152   572 13015 E FA      : Missing google_app_id. Firebase Analytics disabled. See https://goo.gl/NAOOOI
04-03 19:51:19.097 13165 13191 V FA      : Deferring to Google Analytics for Firebase for event data collection. https://goo.gl/J1sWQy
04-03 19:51:19.307 13165 13212 I FA      : App measurement disabled via the manifest
04-03 19:51:19.315 13165 13212 I FA      : App measurement initialized, version: 86006

Repro Rate: 100%

I've tried updating all SDKs and Unity, but the problem still remains.

**Comment by barilki** [comment]
Same

**Comment by paulinon** [comment]
Hi folks,

Does your current `google-services.json` file have a value for `mobilesdk_app_id`? Could you try if downloading a new `google-services.json file` makes a difference? If it doesn't, please provide a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of your implementation so we can identify what's causing this to happen.

**Comment by okuar** [comment]
Hello. Thank you for your response. Yes, my google-services.json file contains a mobilesdk_app_id, and I have also tried downloading a new file and placing it both in the Asset and StreamingAsset folders.

I followed the guide described at this link: https://firebase.google.com/docs/unity/setup, and I also tried to build the quickstart app from https://github.com/firebase/quickstart-unity/tree/master/analytics/testapp, but I encountered the same error at the start of the application.

_Failed to read Firebase options from the app's resources. Either make sure google-services.json is included in your build or specify options explicitly._

I tried to specify options explicitly, entering the ApiKey, AppId, ProjectId, etc., directly, and then the error disappeared, but logcat reported _"Firebase Analytics disabled due to missing google_app_id on Android."_ DebugView in the Firebase console could not see either my device or the events sent.

**Comment by okuar** [comment]
HI, I apologize for my complete absence. The issue is still relevant, and
unfortunately, I have not had the opportunity to update the status of this
task or respond to comments recently.

If it's possible to reopen the task, I could provide new information.

I tried the solution that was suggested at
https://github.com/firebase/firebase-unity-sdk/issues/981#issuecomment-2063379635
but this solution does not allow creating a build and causes other errors
during resource packaging.

Currently, I am in communication with Firebase support via email.
I reproduced the same error in an MCVE. It was suggested to increase the
target API version to 33, but this also did not solve the problem.

If it's still possible to reopen the task, I can provide updates on the
task and respond more actively to comments. Apologies if this causes any
inconvenience.

Regards,
Alex
